spec/models/product_specific_rule_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe ProductSpecificRule do
+  let(:api_host) { TradeTariffFrontend::ServiceChooser.api_host }
+  let(:response_headers) { { content_type: 'application/json; charset=utf-8' } }
+
+  describe '.all' do
+    let :json_response do
```

Comment:
```suggestion
    let(:json_response) do
```

Surprised rubocop didn't flap at this